cmd/version.go

```diff
@@ -14,12 +16,29 @@ var versionCmd = &cobra.Command{
 	Long: `
     Print the build information.
     Please provide the output of this command when reporting issues.`,
-	Run: func(cmd *cobra.Command, args []string) {
-		fmt.Println("Build Date:", version.BuildDate)
-		fmt.Println("Git Commit:", version.GitCommit)
-		fmt.Println("Version:", version.Version)
-		fmt.Println("Go Version:", version.GoVersion)
-		fmt.Println("OS / Arch:", version.OsArch)
+	RunE: func(cmd *cobra.Command, args []string) error {
+		log := printer.Logger()
+		log.Debug().Msgf("gathering version information without output of %s", printer.CommandOut)
+		switch printer.CommandOut {
+		case printer.OutJson:
+			results, _ := api.NewResultFromKeyPair([]string{
+				"build_date:", version.BuildDate,
+				"git commit:", version.GitCommit,
+				"version:", version.Version,
+				"go_version:", version.GoVersion,
+				"os_arch:", version.OsArch,
+			})
+			printer.JsonResult("forgeops version", results)
+		case printer.OutText:
+			fmt.Println("Build Date:", version.BuildDate)
+			fmt.Println("Git Commit:", version.GitCommit)
+			fmt.Println("Version:", version.Version)
+			fmt.Println("Go Version:", version.GoVersion)
+			fmt.Println("OS / Arch:", version.OsArch)
```

Comment:
`fmt`?? So why not use the printer here?

Reply:
This was part of the original command before the printer, I didn't pay any attention to it when I updated.